# class cpuData:
#     float32 cpu_usage_percent
#     float32 cpu_freq_min
#     float32 cpu_freq_max


float32 cpu_usage_percent
float32 cpu_freq_current
float32 cpu_freq_min
float32 cpu_freq_max
float32 virtualMemory_percent

float32[] process_usage_list
string[] process_name_list
